__tests__/test-data.test.js

```diff
@@ -133,3 +133,63 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with status code 200 with valid request", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("returns array of correct length filled with objects containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
```

Comment:
You know what your article_id is, change this to check that it is 1 rather than any number